pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.towncrier]
+    package = "setuptools"
+    package_dir = "setuptools"
+    filename = "CHANGES.rst"
+    directory = "changelog.d"
+    title_format = "v{version}"
+    issue_format = "#{issue}"
+    template = "towncrier_template.rst"
+    underlines = ["-"]
+
+    [[tool.towncrier.type]]
+        directory = "deprecations"
```

Comment:
Is there a good reason why this is plural while the "directory" entry for Changes is the singular "change"?

Reply:
No, good catch.